src/serializers/pot.serializer.ts

```diff
@@ -1,4 +1,6 @@
 import { SerializerInterface } from './serializer.interface';
+/// <reference path="../typings/po2json"/>
```

Comment:
Can you move the typings to src/declarations.d.ts instead? Inline typings hurts my eyes :p